landoui/template_helpers.py

```diff
@@ -32,6 +32,30 @@ def is_user_authenticated() -> bool:
     return helpers.is_user_authenticated()
 
 
+@template_helpers.app_template_global()
+def is_treestatus_user() -> bool:
+    if not is_user_authenticated():
+        return False
+
+    try:
+        userinfo = session["userinfo"]
+    except KeyError:
+        return False
+
+    try:
+        groups = userinfo["https://sso.mozilla.com/claim/groups"]
+    except KeyError:
+        return False
+
+    return any(
+        group in groups
+        for group in {
+            "mozilliansorg_treestatus_admins",
+            "mozilliansorg_treestatus_users",
+        }
+    )
```

Comment:
nit: This might be more readable as a set intersection, e.g.,
```suggestion
    valid_groups = {
        "mozilliansorg_treestatus_admins",
        "mozilliansorg_treestatus_users",
    }
    return bool(valid_groups.intersection(groups))
```